src/main/java/uk/gov/pay/api/app/config/PublicApiConfig.java

```diff
@@ -2,7 +2,7 @@
 
 import com.bendb.dropwizard.redis.JedisFactory;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.cache.CacheBuilderSpec;
+import com.github.benmanes.caffeine.cache.CaffeineSpec;
```

Comment:
What’s this change for?

Reply:
Dropwizard uses `CaffeineSpec` instead of `CacheBuilderSpec` in v2.0.0 for `CachingAuthenticator`

https://github.com/dropwizard/dropwizard/blob/master/dropwizard-auth/src/main/java/io/dropwizard/auth/CachingAuthenticator.java#L42